src/shared/helpers/generateLink.tsx

```diff
@@ -14,21 +14,39 @@ export const CONTENT_TYPE_TO_ROUTE: { [contentType in Avo.Core.ContentType]: str
 	bundel: RouteParts.Folder,
 };
 
+export function buildLink(route: string, params: { [key: string]: any } = {}) {
+	if (route.includes(':') && (isNil(params) || isEmpty(params))) {
+		const missingParams = route
+			.split('/')
+			.filter(r => r.includes(':'))
+			.join(', ');
+		console.error(`Include following params: ${missingParams}`);
```

Comment:
mss moeten we hier een error throwen en de navigatie dan afbreken
of een toast message tonen, aangezien de gebruiker momenteel zal navigeren naar de homepage zonder error bericht

Reply:
aangezien alle uses van deze functie gebruikt worden met history.push, moet je mss history meegeven als param en dan kan je in de functie de navigatie afbreken als er iets mis gaat met de params. Dan blijft de gebruiker op dezelfde pagina

![image](https://user-images.githubusercontent.com/1710840/68111146-c5759600-feee-11e9-812a-c3c76e4069d5.png)

Reply:
Fixed